### DISCUSSION TITLE
Do overlays update if there's a new rating?

### DISCUSSION THREAD

**Discussion by alexkahler** [meisnate12/Plex-Meta-Manager#1898] [Q&A] [answered]
Do overlays update with new stats if, for example, there are new ratings or if show status has changed (e.g., changed from airing to ended) while the `reapply_overlays` is set to `false`?

Apologies for the newbie questions 😣

**Comment by YozoraXCII** [comment, marked as answer]
Yes, overlays will always update if the criteria has changed

`reapply_overlays` forces the overlay to fully reapply even when no changes have been made, which can cause image bloat.
